Bearded.Utilities/IO/Logger.cs

```diff
@@ -30,20 +30,20 @@ internal Writer(Logger logger, Severity severity)
             }
 
             public void Log<T>(T obj) {
-                logger.AddEntry(new Entry(obj.ToString(), severity));
+                logger.AddEntry(new Entry(obj?.ToString() ?? "null", severity));
```

Comment:
This actually looks like it was a bug - it would have been trivial to blow up the logger by passing in `null`. No longer. Though I am not 100% sure if outputting 'null' is the best fallback in this case. Generally, `null` prints as empty string in string formatting, but I felt that an empty string, or ignoring the `Log()` call entirely would make it hard to figure out that null was logged. Being explicit about it will make it easier for the user to find errors in their code.

Reply:
I am okay with using a null string here.